docs/.vuepress/theme/styles/theme-dark.styl

```diff
@@ -197,11 +197,23 @@ html.theme-dark {
       border-bottom-color $tabsComponentBorderColor
       background $tabsComponentBackgroundColor
 
-      a {
-        color $fontColor !important
+      .tabs-component-tab {
+        a {
+          color $fontColor !important
+
+          &:hover {
+            color $linkFontColor !important
+          }
```

Comment:
Couldn't it be removed?

Reply:
If I remove this style, the `a` won't be styled properly.

Reply:
I'm sorry, I thought about another place in code. I will add new comment.

Reply:
Added: https://github.com/handsontable/handsontable/pull/8326#pullrequestreview-695825744